crates/autopilot/src/decoded_settlement.rs

```diff
@@ -179,41 +179,29 @@ impl From<(Address, U256, Bytes<Vec<u8>>)> for DecodedInteraction {
 #[derive(Debug, Clone)]
 pub struct OrderExecution {
     pub order_uid: OrderUid,
-    pub executed_solver_fee: Option<U256>,
-    pub sell_token: H160,
-    pub buy_token: H160,
-    pub sell_amount: U256,
-    pub buy_amount: U256,
-    pub executed_amount: U256,
     pub signature: Vec<u8>, //encoded signature
```

Comment:
Why do we need signature?

Reply:
Deleted as not needed anymore after we changed the matching order executions.